data/test-asset-layers.yml

```diff
@@ -676,4 +676,46 @@ westminster:
     asset_item: 'street nameplate'
     http_options:
       url: https://tilma.mysociety.org/resource-proxy/proxy.php?https://westminster.staging/46/query?
+gloucester:
+  - - name: alloy
+      http_options:
+        url: "tilma.staging/alloy/layer.php"
```

Comment:
not sure if this matters, but cypress has a list of domains not to allow requests on, and tilma.staging.mysociety.org is in there, but this isn't. Rest of this file uses real staging URL so for consistency?